projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-navigation.service.ts

```diff
@@ -67,6 +72,35 @@ export class IgxPivotGridNavigationService extends IgxGridNavigationService {
         }
     }
 
+    public override handleAlt(key: string, event: KeyboardEvent): void {
+        event.preventDefault();
+
+        let row = this.grid.gridAPI.get_row_by_index(this.activeNode.row);
+        let expansionRowKey = PivotUtil.getRecordKey(row.data, row.data.dimensions[0])
```

Comment:
`row.data.dimensions[0]` will only ever get the first row dimension and its hierarchy. If you have a second dimension with hierarchy you would not be able to toggle it. You can find the correct dimension based on the `activeNode`'s column.